alor-broker/Astras-Trading-UI
# Issue #1341: Added migrations. Fixed badges

Fixes #1341

# Description

Added migrations. Fixed badges

# Testing

Tested during development.
To test locally you need:
1. Run old branch, reset settings. Check settings (dashboard, widgets). Dashboard should have old instruments selection (yellow....). Widgets should have old badges
2. Run this branch. Use the same account. Check settings. Dashboard should have new instruments selection (hex colors.). Widgets should have new badges

# Risk

No major changes

# Do you agree with those?
